rhscl-postgresql-apb/README.md

```diff
@@ -0,0 +1,25 @@
+rhscl-postgresql-apb
+======================
+
+An apb for deploying [PostgreSQL](https://www.postgresql.org).
+
+## What it does
+* Deploys 1 postgresql container.
+
+## Requirements
+
+## Parameters
+* NAMESPACE: Optional, default 'postgresql-demo-apb', Namespace to deploy the cluster in.
+* POSTGRESQL_DATABASE, default 'admin', Postgresql database name.
+* POSTGRESQL_PASSWORD, default is a randomly generated string, Postgresql database password.
```

Comment:
Is the default way we're encouraging this passing it in as environment variables or passing it in as `--extra-vars` to the command?